pkgs/os-specific/linux/zfs/default.nix

```diff
@@ -215,8 +215,8 @@ in {
   # to be adapted
   zfsStable = common {
     # check the release notes for compatible kernels
-    kernelCompatible = kernel.kernelAtLeast "3.10" && kernel.kernelOlder "5.16";
-    latestCompatibleLinuxPackages = linuxPackages_5_15;
+    kernelCompatible = kernel.kernelAtLeast "3.10" && kernel.kernelOlder "5.15";
+    latestCompatibleLinuxPackages = linuxPackages_5_14;
```

Comment:
What is this for? I don't see how it is used.

Reply:
https://github.com/NixOS/nixpkgs/pull/126980